src/worker-error-reporting.js

```diff
@@ -0,0 +1,72 @@
+/**
```

Comment:
Not sure adding another directory layer is worth it for a single file. I'll do this if we add more shared worker code.